### ISSUE TITLE
It would be nice to have a way to disable false-positives

### ISSUE THREAD

**Issue by jeanraymonddaher** [PaulTaykalo/swift-scripts#18]
The script is currently very useful since its well maintained, does the job and is extremely simple to integrate. 

You could add this to the installation 
1- Place the file inside your project folder directory
2- Call it via $SRCROOT/unused.rb xcode

Given that there are a lot of false positives like mentioning the word protocol in a comment or others, it would be nice to have a way like swift-lint to disable specific warnings. 

ex unused-disable-this which would disable the warning on the specific line 
unused-disable-all which would disable warnings for all the class

What do you think about it ?

Awesome project btw

**Comment by PaulTaykalo** [comment]
@PaulTaykalo I don't think that this needs to be done if we take in account, how this check is implemented.
I don't want to build a castle on the sand :)

The way how is it implemented, doesn't actually work at big scale.
I would rather transition to some SourceKit/SwiftSyntax solution. 
But, unfortuneately, there's no time for that ATM :)

If you have solution that will work, I don't mind from having a PR for that :)
